linkerd/addr/fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+
+target
+corpus
+artifacts
```

Comment:
nit, take it or leave it: could we perhaps add something like
```gitignore
**/fuzz/target
**/fuzz/corpus
**/fuzz/artifacts
```
to the repository's root gitignore, instead? that way, if we add additional fuzzers in the future, we don't have duplicate this gitignore into all of them.

Not a huge deal, just a thought.

Reply:
When creating the fuzzers the initial set up for a given fuzz crate is created by way of `cargo fuzz init`, which creates the folder structure in each of the fuzzer crates. As such, although I agree that it would be annoying if we had to duplicate the files - in this case the files are auto-generated. Because they are auto-generated I probably lean towards keeping them so we won't have to remove auto-generated code when creating new fuzzers in the future.

Reply:
Ah, okay, I hadn't realized that these were automatically generated...that's cool! In that case, never mind!

It might be nice if `cargo fuzz init` included a comment indicating that the file was generated by `cargo fuzz init`...maybe I should open an issue upstream?

Reply:
I think that's a great idea! I can do a PR in the near future on their repo with an implementation of it!